kpops/components/base_components/kafka_connector.py

```diff
@@ -181,12 +182,15 @@ async def deploy(self, dry_run: bool) -> None:
 
     @override
     async def destroy(self, dry_run: bool) -> None:
+        """Delete Kafka Connector (Source/Sink) from the Kafka connect cluster."""
         await self.handlers.connector_handler.destroy_connector(
             self.full_name, dry_run=dry_run
         )
 
     @override
     async def clean(self, dry_run: bool) -> None:
+        """Delete Kafka Connector. If schema handler is enabled, then remove schemas. Delete all the output topics."""
+        await self.destroy(dry_run)
```

Comment:
shouldn't it suffice to call `super().clean()` here?

Reply:
No, because we are not calling destroy in the superclass.

Reply:
Now I have changed the logic and called destroy inside of `PipelineComponent` there, so the `super().clean()` is sufficient.